proxy/proxy.example.yml

```diff
@@ -0,0 +1,55 @@
+server:
+  applicationConnectors:
+    - type: http
+      port: ${OPENLINEAGE_PROXY_PORT:-8080}
+  adminConnectors:
+    - type: http
+      port: ${OPENLINEAGE_PROXY_ADMIN_PORT:-8081}
+  # Enables access logs formatted as JSON
+  # requestLog:
+  #   appenders:
+  #     - type: console
+  #       layout:
+  #         type: access-json
+
+logging:
+  # Levels: TRACE, DEBUG, INFO, WARN, ERROR, FATAL, ALL, OFF
+  level: ${LOG_LEVEL:-INFO}
+  appenders:
+    - type: console
+      # Enables app logs formatted as JSON
+      # layout:
+      #   type: json
+
+#
+# This name is used to identify the lineage source in the different distribution mechanisms.  If can be used in further
+# routing of the lineage events without needing to unpack the event contents. In this example, it identifies that the
+# lineage event has gone through the proxy backend.  However, if a different proxy backend is serving each engine that
+# is producing lineage events, this value can be set up to represent the source engine.
+#
+lineageSourceName: openLineageProxyBackend
+
+#
+# Set to false to prevent lineage events being printed to stdout.
+#
+consoleLog : true
```

Comment:
We can eliminate the need for this flag, if by default, we [configure the console](https://github.com/OpenLineage/OpenLineage/blob/wip/proxy/proxy/src/main/java/io/openlineage/proxy/ProxyConfig.java#L26) lineage stream. Or, the user appends the type to the `lineageStreams` config (see below)